src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.windows.cs

```diff
@@ -182,7 +183,7 @@ void OnFullScreenButtonClick(object sender, RoutedEventArgs e)
 			mediaPlayerElement.Height = displayInfo.Height / displayInfo.Density;
 
 			Children.Clear();
-			fullScreenButton.Content = fullScreenIcon;
+			fullScreenButton.Content = exitFullScreenIcon;
```

Comment:
This was previously `fullScreenIcon`. Is changing it to `exitFullScreenIcon` a typo, or a bug fix?

Reply:
bug fix. They were swapped incorrectly in orginal PR.